pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/grpc/GrpcStatus.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.pbj.runtime.grpc;
+
+/**
+ * Status codes for gRPC. These are added to a {@link GrpcException} when an error occurs.
+ *
+ * @see <a href="https://grpc.github.io/grpc/core/md_doc_statuscodes.html">GRPC Status codes</a>
+ */
+public enum GrpcStatus {
+    /**
+     * The operation completed successfully.
+     */
+    OK, // 0
```

Comment:
Minor: not sure if the numbers in comments are necessary. Should you need to modify the enum, it would be difficult to update them properly. If they have to be equal to the values in the comments then maybe they should be explicitly assigned to a `private final` field via an enum constructor. Or, if this enum is already fixed in terms of its members and their positional numbers, then it's easier to simply mention that explicitly in the javadoc for the enum class itself rather than maintain these individual comments with ordinal numbers.

Reply:
I should also have a unit test that fixes them in place.

Reply:
Good idea.